src/transform/cmvn.cc

```diff
@@ -176,6 +176,27 @@ void FakeStatsForSomeDims(const std::vector<int32> &dims,
   }
 }
 
-
-
+bool AccCmvnStatsWrapper(std::string utt,
```

Comment:
I used this in the old version of generate-random-cmn-offsets.cc, but I am no longer use this function.
Do you think should I move it back?